samples/gibbs_ensemble/gibbs_ensemble_client.py

```diff
@@ -0,0 +1,104 @@
+#
+# Copyright (C) 2013-2019 The ESPResSo project
+#
+# This file is part of ESPResSo.
+#
+# ESPResSo is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ESPResSo is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Client part of the Gibbs ensemble simulation. This script handles the
+simulation boxes and communicates the energies to the host. The Monte-Carlo
+part of the simulation is done by the :file:`gibbs_ensemble_socket.py` script.
+"""
+
+import espressomd
```

Comment:
it's a matter of taste, but good practice is to first import external modules and after a blank the local ones

Reply:
there's a tool called ```isort``` for that: https://github.com/pycqa/isort/